README.md

```diff
@@ -185,79 +185,105 @@ import (
 
 func Test_Something_With_Docker(t *testing.T) {
   opts := &docker.DockerClusterOptions{
-    ImageRepo: "hashicorp/vault",
+    ImageRepo: "hashicorp/vault", // or "hashicorp/vault-enterprise"
     ImageTag:    "latest",
   }
   cluster := docker.NewTestDockerCluster(t, opts)
   defer cluster.Cleanup()
+  
+  client := cluster.Nodes()[0].APIClient()
+  _, err := client.Logical().Read("sys/storage/raft/configuration")
+  if err != nil {
+    t.Fatal(err)
+  }
+}
+```
+
+Or for Enterprise:
+
+```go
+import (
+  "testing"
+  "github.com/hashicorp/vault/sdk/helper/testcluster/docker"
+)
+
+func Test_Something_With_Docker(t *testing.T) {
+  opts := &docker.DockerClusterOptions{
+    ImageRepo: "hashicorp/vault-enterprise",
+    ImageTag:  "latest",
+	VaultLicense: licenseString, // not a path, the actual license bytes
+  }
+  cluster := docker.NewTestDockerCluster(t, opts)
+  defer cluster.Cleanup()
 }
 ```
 
-Here is a more realistic example.  This test should be run with VAULT_BINARY pointing
-at a local Linux vault binary on disk that you wish to run inside the docker containers. 
-If it's an Enterprise binary, you'll also need to set VAULT_LICENSE_CI, or the
-DockerClusterOptions field VaultLicnese. 
+Here is a more realistic example of how we use it in practice.  DefaultOptions uses 
+`hashicorp/vault`:`latest` as the repo and tag, but it also looks at the environment
+variable VAULT_BINARY. If populated, it will copy the local file referenced by
+VAULT_BINARY into the container. This is useful when there is no published image
+containing the vault binary you want to test.
```

Comment:
```suggestion
VAULT_BINARY into the container. This is useful when testing local changes.
```
Very much optional. Your version is more technically accurate but my guess is this says more directly what readers would need to infer from the sentence otherwise.